exchange/exchange-ps/exchange/mailboxes/Restore-RecoverableItems.md

```diff
@@ -227,7 +227,8 @@ Accept wildcard characters: False
 ```
 
 ### -MaxParallelSize
-The MaxParallelSize paramater controls the maximum number of parallel threads restoring. Higher numbers of parallel threads running will typically increase the speed of Restore-RecoverableItems. Valid values are integers from 1 to 10.
+The MaxParallelSize paramater controls the maximum number of parallel threads restoring. Higher numbers of parallel threads running will typically increase the speed of Restore-RecoverableItems for the multiple mailbox restoring scenario. it cannot work on single mailbox restoring scenario. Valid values are integers from 1 to 10. 
```

Comment:
The MaxParallelSize parameter controls the maximum number of restoring threads in parallel. For multiple mailbox restore scenarios, running more threads in parallel will typically improve the completion time of Restore-RecoverableItems. Parallel threads will not improve single mailbox restore scenarios. 

Valid values: integers from 1 to 10.